### PR TITLE
Assignments/boolean birds

### PR DESCRIPTION
**Pull request by Alexandermasters71**

### CODE REVIEW

File: assignments/boolean-birds/code.js [hunk 1]
```diff
@@ -16,3 +16,52 @@
...
+
+const eatsNuts = (isSpottedBird,isRedBird) => {
+  return ! isRedBird || isSpottedBird
+}
+
+const eatsMice = (isSpottedBird,isRedBird) => {
+  return  ! isRedBird || ! isSpottedBird
+}
+
+const isRed = (redburd) => {
```
Comment (gigamonkey) [inline assignments/boolean-birds/code.js:53]: As a style note, it's worth thinking about the names of arguments. `redburd` in addition to the cutesy spelling, kinda implies that the bird is red. Which obviously is the question this function is trying to answer. So I'd expect the variable name to be something like `name`, `species`, or `bird` to give some kind of indication what kind of value we are expecting. It's not a big deal in a small function like this but using good names is one of the most powerful tools we have to make our code understandable so it's good to get in the habit early of always spending the time to come up with good names.

**Review by gigamonkey — CHANGES_REQUESTED** [review]
Please fix the argument names in the `isRed` and related functions. And also run the "Format document" command I showed you guys to clean up some minor formatting issues. Then I'll re-review and no doubt approve this.

**Comment by Alexandermasters71** [comment]
@gigamonkey I redid the 'redburd' thing

**Review by gigamonkey — APPROVED** [review]
Looks good. Ship it!

**Pull request merged**